shared/team-building/container.tsx

```diff
@@ -566,7 +570,14 @@ const mergeProps = (
         ],
         title,
       }
-    : {}
+    : emptyObj
+
+  const onChangeService = (service: Types.ServiceIdWithContact) => {
```

Comment:
Everywhere else here we are being careful to memoize functions to avoid new functions everytime any prop changes. Maybe we should do that here too?